_posts/2016-06-30-state-in-scalable-architectures.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+name: State-in-scalable-architectures
+title: 'State in scalable architectures'
+date: 2016-06-30 00:20:00 +00:00
+author: Felipe Fernández
+image:
+    src: /assets/img/custom/blog/state.jpg
+    attribution:
+      text: Untitled by Daida Medina
+      href: https://www.instagram.com/p/8TdviFHo-g/?taken-by=thelastfreeusername
+tags:
+- state
+- scalability
+- architecture
+
+---
+
+Handling state is one of the cornerstone of software development. Most of business value derived from software relies on state. Depending on the level of abstraction, state, and the approach to handle it, looks like a complete different problem. In this post we'll see what is common to all of them and how modern scalability needs have pushed the industry to innovative and refreshing approaches.
+
+## Levels of abstraction
+
+[Ted Malaska's presentation](https://www.youtube.com/watch?v=sD-rvRkK_u4) about [Spark](http://spark.apache.org/) serves as inspiration for this post. In this screenshot, Ted shows a list of concepts that we have to understand when learning to code:
+
+<img src="/assets/img/custom/blog/learning-to-code.png" alt="Learning to code" title="Learning to code" class="img img-center img-responsive style-screengrab">
+
+You can stop reading here and spend some time thinking in your conceptions about state at any of those levels. When you're done, come back here and let's try to define more formally what state is.
```

Comment:
There is already a link to the video a couple of lines before. I've understand your point, but this video doesn't have a massive implication in the point of post. It serves as inspiration, but only on that screenshot that I took. The idea of different levels of complexity when learning to code. I'm happy if the reader watch the video, that's why I've included the link, but I wouldn't encourage it as it's not heavily related with the post.